tests/pr_test.go

```diff
@@ -87,3 +88,33 @@ func TestRunUpgradeExample(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestPrivateInSchematics(t *testing.T) {
+	t.Parallel()
+
+	const region = "us-south"
+	const testLocation = "examples/private"
+
+	options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
+		Testing: t,
+		Prefix:  "sm-prv-crt-private",
+		TarIncludePatterns: []string{
+			"*.tf",
+			testLocation + "/*.tf",
+		},
+		ResourceGroup:          resourceGroup,
+		TemplateFolder:         testLocation,
+		Tags:                   []string{"test-schematic"},
+		DeleteWorkspaceOnFail:  false,
+		WaitJobCompleteMinutes: 80,
+		Region:                 region,
+	})
+
+	options.TerraformVars = []testschematic.TestSchematicTerraformVar{
+		{Name: "ibmcloud_api_key", Value: options.RequiredEnvironmentVars["TF_VAR_ibmcloud_api_key"], DataType: "string", Secure: true},
+		{Name: "resource_tags", Value: options.Tags, DataType: "list(string)"},
+	}
```

Comment:
It seems the example supports using an existing SM instance, so is there a reasons why the test is not using our permanent instance? We are clipping SM quotas right now so we need to use our permanent instance when possible,

Reply:
Oh hang on, our permanent instance is not a private only instance so I guess thats why. Hmm, we may need to consider a permanent private only instance to prevent us from hitting our quota, but we can revisit that